### ISSUE TITLE
mobmuplat on newer iPhones not "fullscreen"

### ISSUE THREAD

**Issue by lokkiikkol** [monkeyswarm/MobMuPlat#92]
hi, it would be great to have an option to make patches fullscreen on iPhone XS Max devices and I suppose other iPhones with larger screens. right now, MobMuPlat opens with a grey border on top and bottom. or at the very least make the border color user selectable...

**Comment by lokkiikkol** [comment]
so by looking at an mmp file (or the editor) there needs to be a new setting for canvasType, something like tallerPhone with an even extremer aspect ratio

**Comment by monkeyswarm** [comment]
Yes, I'm hoping for a significant revamp in the future that handles a wider range of aspect ratios naturally, instead of forcing a choice of N aspect ratios in the editor.

(Mobmuplat began when there were far fewer aspect ratios on mobile devices)

**Comment by lokkiikkol** [comment]
cool! for a next release (you announced an updated libpd on the list) could
you make the borders black instead of grey? or if you find the time to
implement the flexible aspect ratio thing, even better..
